plan_delegator/src/plan_delegator.cpp

```diff
@@ -206,7 +206,8 @@ namespace plan_delegator
         // TODO UPDATE THE FOLLOWING IF STATEMENT AFTER VANDEN-PLAS release https://github.com/usdot-fhwa-stol/carma-platform/issues/1106
         /////////
         if (latest_trajectory_plan.trajectory_points.size() > 0) {
-            if(latest_trajectory_plan.trajectory_points[0].planner_plugin_name == "InLaneCruisingPlugin"){
+            if(latest_trajectory_plan.trajectory_points[0].planner_plugin_name == "InLaneCruisingPlugin" || 
```

Comment:
Is this needed in case we're starting up and our first maneuver is a CLC maneuver?

Reply:
Yes that's correct, we are not able to move from 0 without this change (if clc is the first maneuver)

Reply:
This TODO block is cleaned up/fixed by my plan delegator refactor PR (https://github.com/usdot-fhwa-stol/carma-platform/pull/1220), which is shown above in lines 194-198. It was integration tested for inlane-cruising, CLC, and stop and wait. So these lines (220-232) can be removed.